static-assets/components/cstudio-admin/mods/content-types.js

```diff
@@ -314,6 +314,9 @@
                                     197,
                                     'saveContentType'
                                   );
+                                  CrafterCMSNext.system.store.dispatch({
+                                    type: 'FETCH_CONTENT_TYPES'
+                                  });
```

Comment:
This doesn't do much since this is inside an iframe. We should look into posting a message for the SiteToolsApp to listen to and refresh them on the main window

Reply:
oks